collection-scripts/gather_ctlplane_resources

```diff
@@ -62,10 +62,12 @@ function gather_ctlplane_resources {
     run_bg /usr/bin/oc -n "${NS}" get network-attachment-definitions -o yaml '>' "${NAMESPACE_PATH}/${NS}/nad.log"
 
     # We make a single request to get lines in the form <pod> <container> <previous_log>
+    # skip pods that are in Pending state (they won't have
+    # a status.containerStatuses field)
     data=$(oc -n "${NS}" get pods -o json | jq -r '
       .items[] |
       .metadata.name as $pod |
-      .status.containerStatuses[] |
+      .status.containerStatuses[]? |
```

Comment:
-1: I think we shouldn't simply "skip" Pods that are in pending state because we're still interested in the `-describe`, where we can get the events (along with the reason they're not starting).
We shouldn't fail, so +1 to fix this query though!